modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -95,7 +95,7 @@
             "content": {
               "application/json": {
                 "example": {
-                  "id": "1012667145V762142"
+                  "id": "1013062086V794840"
```

Comment:
Looks like this PR has the same issue with updated ID that Alex noted in other recent PRs.